_posts/2021-07-16-release-0.9-future.md

```diff
@@ -0,0 +1,39 @@
+
+`wgpu` performance is still a vital target for us, so we have done work on improving the overhead at resource tracking. We've reduced unnecisary overhead through only doing stateful tracking for resources that have complex states. These changes were made from benchmarks of Gecko's WebGPU implemention which showed that tracking was a bottleneck. You can read more about it [#1413](https://github.com/gfx-rs/wgpu/issues/1413).
+
+## wgpu family reunion, relicense, and the future
+
+`wgpu` has had a number of large internal changes which are laying the future for wgpu to be a safe, efficient, and portable api for doing cross-platform graphics.
+
+`wgpu` has been relicensed from `MPL-2.0` to `MIT/Apache-2.0`. Thank you to all 142 people who replied to the issue and made this happen. This relicense is an important change because it allows the posibility of adding backends that target APIs that are behind NDAs.
+
+Thanks to feedback from our friends at the bevy game engine, we launched a initiative to make wgpu easier to contribute to. We moved wgpu-rs back into the `wgpu` repo. This means that PRs that touch both the core crate and the rust bindings no longer need multiple PRs that need to be synchronized. We have already heard from collaborators how much easier the contribution is now that there is less coordination to do. [Read more about the family reunion](https://github.com/gfx-rs/wgpu/milestone/9)
+
+As a part of our family reunion, `0.9` is going to be the last release that will use `gfx-hal` as its hardware abstraction layer. While it has served us well, it has proved to not be at the exact level of abstraction we need. We have started work on a new abstraction layer called `wgpu-hal`. This new abstraction has already had Vulkan, Metal, and GLES ported, with DX12 landed in an incomplete state, and DX11 to come soon. To learn more about this transition, [you can read the whole discussion](https://github.com/gfx-rs/gfx/discussions/3768).
+
+Finally, we have brand new testing infrastructure that allows us to automatically test across all backends and all adapters in the system. Included in our tests are image comparison tests for all of our examples and the beginnings of feature tests. We hope to expand this to cover a wide variety of features and use cases. We will be able to run these tests in CI on software adapters and our future goal is to setting up a distributed testing network so that we can automatically test on a wide range of adapters. This will be one important layer of our in-depth defences, ensuring that wgpu is actually portable and safe. Numerous bugs have already been caught by this new infrastrucutre thus far and it will help us prevent regressions in the future. [Read more about our testing infrastructure](https://github.com/gfx-rs/wgpu/discussions/1611)
+
+## Thank You!
```

Comment:
nice to have this section!
would it make sense to align the capitalization between sections? This one is all capital, but the other one is all lowercase.